be/src/storage/rowset/unique_rowset_id_generator.cpp

```diff
@@ -60,6 +60,20 @@ bool UniqueRowsetIdGenerator::id_in_use(const RowsetId& rowset_id) const {
 }
 
 void UniqueRowsetIdGenerator::release_id(const RowsetId& rowset_id) {
+    // The RowsetId a string of length 48. To reduce memory consumption,
+    // only the high 16-bit string, like "0200000000000006", stored in the unordered_set.
+    // Upon restart, the start point will be reset as zero. The number may
+    // be overlap with the rowsets compacted. After compaction, the high-16 string
+    // will be removed from the unordered_set. The Rowset GC daemon will remove the
+    // files generated by the load in progress.
+
+    // release_id should check backend_uid first to avoid the startup problem.
```

Comment:
The code is self-explained, no need to write these comments

Reply:
OK